.github/workflows/codeigniter.yml

```diff
@@ -2,7 +2,11 @@
 name: Testing Kalkun
 on:
   push:
-    branches: [ "*" ]
+    branches: 
+      - master
+      - devel
+      - 'release/**'
```

Comment:
In some cases (don't remember exactly which one) the "/" is not liked by git. Could we use something else instead? underscore? dot? dash? whatever

Reply:
Sure